test/check-paths.sh

```diff
@@ -50,8 +50,8 @@ exactly 1 "direct usage of decls_repository.get_decl()" 'decls_repository\.get_d
 exactly 21 "direct loc set" -P '(\t|;|\.)loc\s*=(?!=)'
 exactly 0 "magic number mouse opacity set" -P 'mouse_opacity\s*=\s*[0-2]'
 exactly 1 "magic number density set" -P '\bdensity\s*=\s*[01]'
-exactly 0 "magic number anchored set" -P 'anchored\s*=\s*[01]'
-exactly 7 "magic number opacity set" -P 'opacity\s*=\s*[01]'
+exactly 0 "magic number anchored set" -P '\banchored\s*=\s*[01]'
+exactly 8 "magic number opacity set" -P '\bopacity\s*=\s*[01]'
```

Comment:
```suggestion
exactly 7 "magic number opacity set" -P '\bopacity\s*=\s*[01]'
```
The point of using `\b` is to avoid false-positives, so there's no need to increment it from 7 to 8 anymore.

Reply:
I know, per Discord I wanted to see what the actual output of the test was before I changed it.